packages/angular/projects/ui-angular/src/lib/primitives-poc/password-field/password-field-component/password-field.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="amplify-flex amplify-field amplify-textfield">
+  <!-- Label -->
+  <label [for]="inputId" *ngIf="!labelHidden">
+    <ng-content select="[amplify-label]"> </ng-content>
+  </label>
+
+  <!-- Description Text -->
+  <ng-content select="[amplify-description-text]"></ng-content>
+
+  <!-- Actual Input -->
+  <div
+    class="amplify-flex amplify-field-group amplify-field-group--horizontal"
+    data-orientation="horizontal"
+  >
```

Comment:
Can we use `amplify-flex` that you created in #2352? We can do so after we sync this branch with `poc/angular-primitives/alpha`.

Reply:
I will this once flex is on the alpha.